.github/workflows/pr-workflow.yml

```
@@ -125,25 +150,47 @@
         env:
           REGION: ap-northeast-2
           ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           ENV: dev
           SERVICE: main-api
           DOMAIN: api.dev.ratel.foundation
-          ECR_NAME: ratel.foundation/ratel/main-api
+          ECR_NAME: main-api
+          COMMIT: pr-${{ github.sha }}
+          MIGRATE: true
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1009:info:6:1: The mentioned syntax error was in this simple command

(shellcheck)